.github/workflows/lint-go.yaml

```diff
@@ -57,6 +57,24 @@ jobs:
           skip-cache: true
           args: "--out-${NO_FUTURE}format colored-line-number --verbose --modules-download-mode=vendor"
 
+  staticcheck:
+    name: Lint source code using staticcheck (to detect duplicate package imports)
```

Comment:
and change the name here to `Lint Source Code staticcheck`
these names appear in dashboards so we do not want to bloat them if possible.

Thanks